src/50_Sample_AMPs/Live_Blog.html

```diff
@@ -0,0 +1,142 @@
+<!---{
+  "preview": true,
+  "draft": true,
+  "experiment": true,
+  "component": "amp-live-list"
+  }--->
+<!--
+  #### Introduction
+
+This is a sample template for a live blogs in AMP. It demonstrates the usage of AMP components which works well in live blogs.
```

Comment:
Is this sample really about the components?  "It demonstrates the usage of AMP components which works well in live blogs."